drf/university/views.py

```diff
@@ -1,14 +1,35 @@
 from .serializers import UniversitySerializer
+from school.serializers import SchoolSerializer
 from .models import University
 from rest_framework.generics import (ListCreateAPIView,
-                                     RetrieveUpdateDestroyAPIView)
+                                     RetrieveUpdateDestroyAPIView,
+                                     ListAPIView)
 
 
 class UniversityListCreate(ListCreateAPIView):
     queryset = University.objects.all()
     serializer_class = UniversitySerializer
 
 
-class UniversityRetrieveUpdateDestroy(RetrieveUpdateDestroyAPIView):
+class UniversityDetail(RetrieveUpdateDestroyAPIView):
     queryset = University.objects.all()
     serializer_class = UniversitySerializer
+
+
+class UniversitySchoolsList(ListCreateAPIView):
+    serializer_class = SchoolSerializer
+
+    def get_queryset(self):
+        university = University.objects.get(
```

Comment:
You can write something like:
```
return University.objects.filter(pk=self.kwargs.get(self.lookup_field)).select_related('school')
```

Reply:
Get queryset should return queryset not single object. With _get_ you return single object.